README.rst

```diff
@@ -20,33 +20,238 @@
+
+Developer Installation
+``````````````````````
 
-Two main installation options exist for Landlab. Most people will likely want to
-`install the conda package <https://landlab.readthedocs.io/en/latest/install/index.html>`_.
 Individuals interested in modifying the Landlab source code should follow the
-`developer installation instructions <https://landlab.readthedocs.io/en/latest/development/install/index.html>`_.
+`developer installation instructions <https://landlab.readthedocs.io/en/latest/development/install/index.html>`_
+which describe cloning the source code, creating a conda environment for
+development, compiling, and testing the code.
+
+In short, clone the repository, navigate to the top level directory, and
+the following commands:
+
+.. code-block:: bash
+
+    $ conda env create --file=environment-dev.yml
+    $ conda activate landlab_dev
+    $ python setup.py develop
+
+How do I verify I've installed Landlab correctly?
+-------------------------------------------------
+
+Landlab uses pytest to discover and run tests. Presuming you have used a source
+code installation with the above conda environment, you will be able to test
+your install with
+
+.. code-block::
+
+    $ pytest
+
+from within the ``landlab_dev`` conda environment. Additional instructions
+can be found `here <>`_.
+
+What are Landlab's dependencies?
+--------------------------------
+
+The core package dependencies are specified by ``requirements.txt`` and used
+by ``setup.py``. There are some additional dependencies that exist for
+running the notebooks or modifying the source code and testing.
+
+Details of how we structure our dependencies can be found `here <>`_.
+
+How do I learn more about Landlab?
+----------------------------------
+
+Our documentation is hosted on ReadTheDocs at https://landlab.readthedocs.io/.
+This includes a User Guide and API reference.
+
+Hobley et al. (2017) describes the design of Landlab.
+
+`Hobley, D. E. J. <https://www.earth-surf-dynam.net/5/21/2017/>`__, Adams,
+J. M., Nudurupati, S. S., Hutton, E. W. H., Gasparini, N. M., Istanbulluoglu,
+E. and Tucker, G. E., 2017, Creative computing with Landlab: an open-source
+toolkit for building, coupling, and exploring two-dimensional numerical models
+of Earth-surface dynamics, Earth Surface Dynamics, 5, p 21-46,
+10.5194/esurf-5-21-2017.
+
+Are there any examples of using Landlab I can look at?
+------------------------------------------------------
+
+The Landlab package contains a directory called ``notebook`` which contains
+Jupyter notebooks describe core concepts and give examples of using components.
+The file ``landlab/notebooks/welcome.ipynb`` provides a table of contents to
+the notebooks and is the recommended starting place. To launch an instance of
+Binder and explore the notebooks click
+`here <https://mybinder.org/v2/gh/landlab/landlab/release?filepath=notebooks/welcome.ipynb>_`.
+
+In addition there are a set of curated teaching notebooks located in
```

Comment:
"In addition there are a set of notebooks curated to teach physical processes located in"

Reply:
fixed.